run.sh

```diff
@@ -1,4 +1,15 @@
-source env/bin/activate;
+function setup_err(){
+  echo "Need to run setup.sh"
+  exit 1
+}
+source env/bin/activate || setup_err
+if [ "$#" -lt "2" ] ; then
+  echo "Usage: ./run.sh python3 evofinder.py <input python file> <target_line_num> <example_file_1> <example_file_2> ..."
+  exit 1
+elif [ ! $1  = "python3" ] || [ ! $2 = "evofinder.py" ] ; then 
+  echo "Usage: ./run.sh python3 evofinder.py <input python file> <target_line_num> <example_file_1> <example_file_2> ..."
+  exit 1
+fi
```

Comment:
I don't think we need this if condition at all. This requires us to maintain a lot of different argument-handling logic, which is unnecessary. Let's just let users pass anything they want to run.sh. For example, when I tested your run.sh script, I didn't use the format above. Instead I did:

`./run.sh ./evofinder ...`

Reply:
Shall I delete the argument count check too?

Reply:
Maybe just change it to print out a message if no arguments are supplied, simply indicating they should do `./run.sh <some_shell_command>` or something?